lib/sqlalchemy/sql/elements.py

```diff
@@ -1831,7 +1845,7 @@ def _dedupe_anon_label_idx(self, idx: int) -> str:
             return self._dedupe_anon_tq_label_idx(idx)
 
     @property
-    def _proxy_key(self):
+    def _proxy_key(self) -> Optional[str]:
```

Comment:
Mypy reports incompatible signature with supertype `ColumnElement`. The signature is the same, tried changing the `@property` to `@HasMemoized.memoized_attribute` and that removed the error. Will look closer at decorated overrides that currently raise errors.

Reply:
Maybe it could be solved with typing the decorator `@HasMemoized.memoized_attribute` someway like this: https://stackoverflow.com/a/65613529/11283875